tools/restart_lipstick.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+nohup sh -c "pkill jolla-settings;systemctl --user --no-block restart lipstick.service" 1>/dev/null 2>/dev/null &
```

Comment:
Why do you kill `jolla-settings` in first place?
It seems to me that `systemctl --user --no-block restart lipstick.service` is enough. Am I missing something?